internal/databases/postgres/configure.go

```diff
@@ -58,3 +59,16 @@ func getStopBackupTimeoutSetting() (time.Duration, error) {
 
 	return timeout, nil
 }
+
+func getPgTimeoutSetting() (time.Duration, error) {
+	if !viper.IsSet(conf.PgTimeout) {
+		return 0, nil
+	}
+
+	timeout, err := conf.GetDurationSetting(conf.PgTimeout)
+	if err != nil {
+		return 0, err
+	}
+
+	return timeout, nil
```

Comment:
could be simplified to `return conf.GetDurationSetting(conf.PgTimeout)`

Reply:
fixed